docs/central-users.rst

```diff
@@ -19,67 +19,94 @@ Central features Role-based User permissioning. In the current release of Centra
+| Edit & Review                    | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Comment                          | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| View & Download                  | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| OData Access                     | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| **Project Entity Lists**                                                                             |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Create                           | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| List All                         | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Add Properties                   | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Update Settings                  | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| **Project Entities**                                                                                 |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Create from Submission           | x             | x               |                | x              |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Create Directly                  | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| View in Linked Forms             | x             | x               | x              | x              |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| | View & Download Anywhere,      | x             | x               | x              |                |
+| | Not Just in Linked Forms       |               |                 |                |                |
```

Comment:
Do actions need to be title-case? I feel like it doesn't work particularly well for longer actions like this one.

Reply:
I don't think they should be. I removed it in a commit.